static/scripts/onboarding/onboarding.ts

```diff
@@ -32,92 +35,87 @@ let encryptedValue = "";
 
 interface ConfLabel {
   name: string;
-  weight: number;
-  value?: number | undefined;
-  target: string;
+}
+
+interface CommandLabel {
+  name: string;
+  enabled: boolean;
+}
+
+interface IIncentive {
+  comment: {
+    elements: Record<string, unknown>;
+    totals: {
+      word: number;
+    };
+  };
+}
+
+interface IControl {
+  label: boolean;
+  organization: boolean;
 }
 
 interface IConf {
-  "chain-id"?: number;
   "private-key-encrypted"?: string;
   "safe-address"?: string;
   "base-multiplier"?: number;
-  "time-labels"?: ConfLabel[];
-  "priority-labels"?: ConfLabel[];
   "auto-pay-mode"?: boolean;
   "analytics-mode"?: boolean;
   "max-concurrent-bounties"?: number;
   "incentive-mode"?: boolean;
+  "evm-network-id"?: number;
+  "price-multiplier"?: number;
+  "issue-creator-multiplier"?: number;
+  "payment-permit-max-price"?: number;
+  "max-concurrent-assigns"?: number;
+  "assistive-pricing"?: boolean;
+  "disable-analytics"?: boolean;
+  "comment-incentives"?: boolean;
+  "register-wallet-with-verification"?: boolean;
+  "promotion-comment"?: string;
+  "default-labels"?: string[];
+  "time-labels"?: ConfLabel[];
+  "priority-labels"?: ConfLabel[];
+  "command-settings"?: CommandLabel[];
+  incentives?: IIncentive;
+  "enable-access-control"?: IControl;
 }
 
-const defaultConf: IConf = {
-  "chain-id": 1,
+let defaultConf: IConf = {
```

Comment:
Perhaps it is wise to dynamically load this in from our repository directly via a network request?

https://github.com/ubiquity/ubiquibot/blob/development/ubiquibot-config-default.json